components/time/src/ixdtf.rs

```diff
@@ -556,39 +570,6 @@ impl<A: AsCalendar> ZonedDateTime<A, TimeZoneInfo<models::Full>> {
     /// An RFC 9557 string may contain both a UTC Offset and time zone annotation. This is fine as long as
     /// the time zone parts can be deemed as inconsistent or unknown consistency.
     ///
-    /// ### UTC Offset with IANA identifier annotation
-    ///
-    /// In cases where the UTC Offset as well as the IANA identifier are provided, some validity checks are performed.
```

Comment:
Issue: Do we still perform the validity check in datetime? I think so. Please re-create the test over in the datetime crate, then.